common/build/eslint-config-fluid/README.md

```diff
@@ -8,4 +8,37 @@ It exports the following shared ESLint configs:
 - `recommended`: The recommended config for Fluid Framework packages.
 - `strict`: The strictest config; intended for packages with public facing APIs.
 
+## Changing the lint config
+
+If you want to change the shared lint config (that is, this package), you need to do the following:
+
+1. Make the change in the config.
+2. Publish a pre-release package.
+3. Update the core packages to use the pre-release lint config.
+
+### Tracking lint config changes over time
+
+One question that comes up often when we make changes to our lint config is, "what changed?" This applies even when we
+don't make any changes other than upgrading deps, because the dependency upgrade might include a new rule.
+
+ESLint provides a way to print the config that would apply to a file (`--print-config`), so we use this capability to
+print out the applied config as a JSON file. As we make changes to the config, we can print out the config again and get
+a diff to review as part of a PR -- just like we do with API reports for code changes.
+
+When updating the lint config, run the `build` script for this package and commit the resulting changes.
```

Comment:
sounds like this step should be included in the above section on how to make a change

Reply:
Yup, good point. Will fix.